src/m365/spo/commands/page/PageControl.ts

```diff
@@ -0,0 +1,35 @@
+
+export interface PageControl {
+  id: string;
+  position: Position;
+  emphasis: Emphasis;
+  controlType?: number;
+  displayMode?: number;
+  webPartId?: string;
+  addedFromPersistedData?: boolean;
+  reservedHeight?: number;
+  reservedWidth?: number;
+  webPartData?: WebPartData;
+}
+
+export interface WebPartData {
+  id: string;
+  instanceId: string;
+  title: string;
+  description: string;
+  audiences: any[];
+  serverProcessedContent: null[];
+  dataVersion: string;
+  properties: any[];
+}
+
+export interface Emphasis {
```

Comment:
If the interface doesn't have any properties for us, let's not define it and in line 5 set the type as an empty object `{}`